test/Plutip.purs

```diff
@@ -566,6 +561,37 @@ suite = do
             liftedM "Failed to balance/sign tx" $ balanceAndSignTx ubTx
           submitAndLog bsTx
 
+    test "runPlutipContract: NativeScriptMints" do
+      let
+        distribution :: InitialUTxOs
+        distribution =
+          [ BigInt.fromInt 5_000_000
+          , BigInt.fromInt 2_000_000_000
+          ]
+      runPlutipContract config distribution \alice -> do
```

Comment:
Could you please also demonstrate moving the tokens around, e.g. sending to self?